cookbook/workflows/team_workflow.py

```diff
@@ -0,0 +1,129 @@
+from agno.workflow import Workflow
+
+class TeamWorkflow(Workflow):
+    def __init__(self):
+        super().__init__()
+        self.name = "Team Workflow"
+        self.description = "A workflow for a team of agents"
+        """Please install dependencies using:
+pip install openai newspaper4k lxml_html_clean agno
+"""
+
+import json
+from textwrap import dedent
+from typing import Iterator
+
+import httpx
+from agno.agent import Agent, RunResponse
+from agno.tools.newspaper4k import Newspaper4kTools
+from agno.utils.log import logger
+from agno.utils.pprint import pprint_run_response
+from agno.workflow import Workflow
+
+from agno.agent import Agent
+from agno.models.openai import OpenAIChat
+from agno.team.team import Team
+from agno.tools.hackernews import HackerNewsTools
+from agno.tools.exa import ExaTools
+
+
+class HackerNewsReporter(Workflow):
+    description: str = (
+        "Get the top stories from Hacker News and write a report on them."
+    )
+
+   
+    reddit_researcher = Agent(
+        name="Reddit Researcher",
+        role="Research a topic on Reddit",
+        model=OpenAIChat(id="gpt-4o"),
+        tools=[ExaTools()],
+        add_name_to_instructions=True,
+        instructions=dedent("""
+            You are a Reddit researcher.
+            You will be given a topic to research on Reddit.
+            You will need to find the most relevant posts on Reddit.
+        """),
+    )
+
+    hackernews_researcher = Agent(
+        name="HackerNews Researcher",
+        model=OpenAIChat("gpt-4o"),
+        role="Research a topic on HackerNews.",
+        tools=[HackerNewsTools()],
+        add_name_to_instructions=True,
+        instructions=dedent("""
+            You are a HackerNews researcher.
+            You will be given a topic to research on HackerNews.
+            You will need to find the most relevant posts on HackerNews.
+        """),
+    )
+
+
+    agent_team = Team(
```

Comment:
Lets rather use a `route` or `coordinate` example, it is simpler. I think it can also be the hackernews team though, it fits.